File: frontend/src/model/ParentDto.ts
import UserShortDto from '@/model/UserShortDto.ts';

interface ParentDto {
  id: string;
  login: string;
  firstName: string;
  lastName: string;
  role: string;
  schoolId: string;
  student: UserShortDto;
}

export default ParentDto;

File: Application/Dto/ParentDto.cs
using Domain.Entities;

namespace Application.Dto;

public class ParentDto
{
    public int Id { get; set; }
    public string Login { get; set; } = "";
    public string FirstName { get; set; } = "";
    public string LastName { get; set; } = "";
    public UserRole Role { get; set; }
    public int SchoolId { get; set; }
    public UserShortDto Student { get; set; } = new();
}